app/helpers/script_helper.rb

```diff
@@ -16,12 +16,19 @@ def render_javascript_pack_once_tags(...)
       concat javascript_assets_tag
       @scripts.each do |name, (url_params, attributes)|
         asset_sources.get_sources(name).each do |source|
+          crossorigin = true if local_crossorigin_sources?
```

Comment:
we should be able to directly assign this right? I'd go a step further an inline the entire thing, but that would make us have to wrap line 23 below
```suggestion
          crossorigin = local_crossorigin_sources?
```

Reply:
ah looks like the [source code](https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#:~:text=%3B%20as%3Dscript%22-,preload_link%20%2B%3D%20%22%3B%20crossorigin%3D%23%7Bcrossorigin%7D%22%20unless%20crossorigin.nil%3F,-preload_link%20%2B%3D%20%22%3B%20integrity) checks for "not nil" and "true"

maybe it would be easier if we updated `local_crossorigin_sources?` to match that expected format?

```ruby
# Fake boolean to match expected values for `:crossorigin` option in `javascript_include_tag`
# @see ActionView::Helpers::AssetTagHelper#javascript_include_tag
# @return [true, nil]
def crossorigin
  true if Rails.env.development? && ENV['WEBPACK_PORT'].present?
end
```

Reply:
I tinkered with this a bit, and landed toward a small revision to use `presence` in 4e23c1eee0.

Partly for micro-optimization, since the value is referenced twice and would prefer to avoid evaluating that logic twice. But moreso because the `true` / `nil` vs. `true` / `false` is less to do with some internal logic specific to `javascript_include_tag`, and moreso about the [`crossorigin` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin) itself, where `false` would apply an empty string attribute value, which is significant since it's an [HTML boolean attribute](https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML).